packages/moonstone/DaySelector/DaySelectorDecorator.js

```diff
@@ -91,9 +90,11 @@ const DaySelectorDecorator = hoc((config, Wrapped) => {
 			everyWeekendText: PropTypes.string,
 
 			/**
-			 * Called when an item is selected. The first parameter will be an object containing a
-			 * `selected` member, containing the array of numbers representing the selected days, zero
-			 * indexed.
+			 * Called when an day is selected or unselected.
+			 *
+			 * The event payload will be an object with the following members:
```

Comment:
@webOS101 what's the proper way to document event payloads if we don't want to use `@typedef`?